### ISSUE TITLE
First constant array is marked as problem

### ISSUE THREAD

**Issue by Enchillada411** [genius257/vscode-autoit#48]
It looks like there is an issue with constant arrays.
Only the first occurrence of an constant array is marked as problem.

```
Const $constVar = 2
Const $constArray[2] = [1, 2]
Const $constArray2[2] = [1, 2]
```

Problem:
Expected " ", "=", "\t", or [0-9a-zA-Z_] but "[" found.

[{
	"resource": "/C:/Users/GlobalConstArray.au3",
	"owner": "_generated_diagnostic_collection_name_#0",
	"severity": 8,
	"message": "Expected \" \", \"=\", \"\\t\", or [0-9a-zA-Z_] but \"[\" found.",
	"startLineNumber": 2,
	"startColumn": 18,
	"endLineNumber": 2,
	"endColumn": 19
}]

**Comment by genius257** [comment]
The parser for AutoIt has the fix for this in [autoit-pegjs v1.2.0](https://github.com/genius257/autoit3-pegjs/releases/tag/1.2.0)
I will upgrade the dependency and run some manual tests, before I'll release a new version

**Comment by genius257** [comment]
@Enchillada411, the issue should now be solved, please verify 😃

**Comment by Enchillada411** [comment]
> @Enchillada411, the issue should now be solved in version 1.5.0 (just released), please verify 😃

Looks good. Thanks a lot for this quick fix.
